org-roam.el

```diff
@@ -163,6 +163,21 @@ Marks from the Latin alphabet."
   :type '(repeat character)
   :group 'org-roam)
 
+(defcustom org-roam-slug-name-replacement-rules
+  '(("[^[:alnum:][:digit:]]" . "_")  ;; convert anything not alphanumeric
+    ("__*" . "_")                    ;; remove sequential underscores
+    ("^_" . "")                      ;; remove starting underscore
+    ("_$" . ""))                     ;; remove ending underscore
+  "Replacement rules applied after Unicode normalization to get the slug.
+
+By default the following rules are applied:
+1. All not alphanumeric characters are replaced by underscores
+2. Sequential underscores are replaced by a single underscore
+3. Starting underscores are removed
+4. Ending underscores are removed"
```

Comment:
This is great. So much passive voice, though: are applied, are replaced (2×), are removed (2×). I would go with apply, replace, and remove.

Reply:
I used passive voice because something is done with something. I will check the other docs and revise this text if necessary.